README.md

```diff
@@ -1,6 +1,6 @@
-<p align="center">
+<h1 align="left">
```

Comment:
I would prefer to keep the banner centered as it currently is. Another 3rd party example of this https://github.com/Textualize/rich